### PR TITLE
python-sqlmodel: New package

### PR DESCRIPTION
**Pull request by podsvirov**
~~Based on https://github.com/tiangolo/sqlmodel/pull/563.~~

*UPDATE*: Good news. Porting to `SQLAlchemi 2.0` done there https://github.com/tiangolo/sqlmodel/pull/700.

### CODE REVIEW

File: mingw-w64-python-sqlmodel/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Maintainer: Konstantin Podsvirov <konstantin@podsvirov.pro>
+
+_realname=sqlmodel
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=0.0.8
```
Comment (Biswa96) [inline mingw-w64-python-sqlmodel/PKGBUILD:6]: Should pkgver be generated from git commit using pkgver function?

Comment (podsvirov) [reply mingw-w64-python-sqlmodel/PKGBUILD:6]: I don't think it's necessary. I hope this PR will be merged and we will soon see a new version with official `sqlalchemy 2.0` support.

Comment (podsvirov) [reply mingw-w64-python-sqlmodel/PKGBUILD:6]: We can make this PR a draft. I just want to draw attention to the need to add `sqlalchemy 2.0` support for this package in a modern distribution. We already have an [FastAPI](https://packages.msys2.org/base/mingw-w64-python-fastapi), but no integration with database.

**Comment by podsvirov** [comment]
Cc: @lazka, @Biswa96

**Comment by lazka** [comment]
This also has a broken version: `Version: 0`

**Comment by lazka** [comment]
I've created an upstream question (https://github.com/tiangolo/sqlmodel/discussions/701) since bug reports aren't allowed.

**Comment by lazka** [comment]
I guess in this case we have to patch the pyproject.toml

**Comment by podsvirov** [comment]
> I guess in this case we have to patch the pyproject.toml

Done, please review again.

**Pull request merged**